packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -4248,3 +4248,28 @@ class RenderFollowerLayer extends RenderProxyBox {
     properties.add(new TransformProperty('current transform matrix', getCurrentTransform()));
   }
 }
+
+/// Render object which inserts an [AnnotatedRegionLayer] into the layer tree.
+///
+/// See also:
+///
+///   * [Layer.find], for an example of how this value is retrieved.
+///   * [AnnotatedRegionLayer], the layer this render object creates.
+class AnnotatedRegionRenderObject<T> extends RenderProxyBox {
+  /// A value which can be retrieved using [Layer.find].
```

Comment:
The lack of a constructor is very atypical for RenderObject subclasses.

Reply:
Fixed